scripts/publish-release.sh

```diff
@@ -32,6 +32,7 @@ if [ "${RELEASE_ID}" = "null" ]; then
   RELEASE_DATA="{
     \"tag_name\": \"${CIRCLE_TAG}\",
     \"target_commitish\": \"${CIRCLE_SHA1}\",
+    \"discussion_category_name\": \"announcement\",
```

Comment:
is creating a discussion mandatory to triger the build? it is fine, i just don't see why

Reply:
No, it is not required. Updated to make it conditional.